test/e2e/test_tools.py

```diff
@@ -21,8 +20,8 @@ class TestTools(unittest.TestCase):
     password = 'test'
     imgdir = '.'
     sipi = 'http://0.0.0.0:1024'
-    test_onto_file = 'testdata/test-onto.json'
-    test_list_file = 'testdata/test-list.json'
+    test_project_file = 'testdata/test-project-systematic.json'
+    test_project_minimal_file = 'testdata/test-project-minimal.json'
```

Comment:
It is not only important to have a systematic test project, but also to have a minimal one. During my work, I accidentally spotted a bug which was never detected. With the new minimal test project, it would have been detected.